app/src/main/java/com/example/poste/PosteApplication.java

```diff
@@ -40,25 +110,39 @@ public static User getCurrentUser() {
      *
      * @param currentUser The User object representing the currently logged-in user
      */
-    public static void setCurrentUser(User currentUser) {
+    public static void setCurrentUser(com.example.poste.api.poste.models.User currentUser) {
```

Comment:
As a result of above, this can be removed, and anywhere in the code that uses it can be deleted outright (as the user will never need to be set)

Reply:
I will extract this in the sprint 3 so it does trick me when building the MVP.
If I removed it this sprint it will likely bring up merge conflicts.
I already have a US task for this I am assigned to -> https://tree.taiga.io/project/dzemlin-poste/task/145

Reply:
As above:

Please be specific in user stories / tasks so that individual needs aren't forgotten. For instance, specifying *which* code to remove, refactor, etc.

I'd much rather see this here so we aren't changing the same code twice, and since this code is already changing here -- also, merge conflicts are the result of multiple developers working on the same code, so no merge conflict looks to arise from this.

I'll mark this as resolved for now, but have updated the Taiga task. Please consider for the future.